packages/ui-components/src/components/tag-alarm/tag-alarm.scss

```
@@ -0,0 +1,109 @@
+@use 'sass:map';
+
+@import '../../assets/styles/globals';
+
+$icon-sizes: (
+	xxsmall: 12px,
+	xsmall: 16px,
+	small: 18px,
+	normal: 24px,
+	large: 28px
+);
+
+$alarms-severity-colors: (
+	'1': #a71300,
+	'2': #e0a93b,
+	'3': #9e0c96,
+	'4': #305960,
+	'5': #2261d4
+);
+
+@mixin icon-size-theme($size) {
+	kv-icon {
+		--icon-height: #{map.get($icon-sizes, $size)};
+		--icon-width: #{map.get($icon-sizes, $size)};
+	}
+}
+
+@mixin icon-color-theme($severity) {
+	$color: map.get($alarms-severity-colors, $severity);
+
+	kv-icon {
+		--icon-color: #{$color};
+	}
+}
+
+@mixin badge-bg-color-theme($severity) {
+	$color: map.get($alarms-severity-colors, $severity);
+
+	kv-tag {
+		--tag-background-color: #{rgba($color: $color, $alpha: 0.15)};
```

```suggestion
		--tag-background-color: #{rgba($color: $color, $alpha: 0.16)};
```
to match Figma specs